bin/mail/2016-09-27-hoc-volunteer-launch/generate-mailing-list.rb

```diff
@@ -2,5 +2,12 @@
 require_relative '../mailing-common/mailing-list-utils'
 
 contacts = query_subscribed_contacts q: 'kind_s:VolunteerEngineerSubmission2015'
+
+# Add name_s field, copied from name, because name_s is referenced in the template and name is stripped out here:
+# https://github.com/code-dot-org/code-dot-org/blob/cefeb5a0f90b06a65281187ef3ff40b5eb93d83e/bin/mail/send-to-mailing-list#L77
+contacts.values.each do |contact|
+  contact[:name_s] = contact[:name]
+end
```

Comment:
I can't think of any privacy reason in this case where the name is actually used in the email template. 

Generally, the params are [queued up](https://github.com/code-dot-org/code-dot-org/blob/271b2b4a3336d44380e2098f77e24a9d29f2a4f6/lib/cdo/poste.rb#L243-L251) in the `Pegasus.post_deliveries` table. The email is saved in the `contacts` table, and also in the `contact_email` field in `poste_deliveries`, and the `contacts` entry is linked via `contact_id`. Strangely `name` only used in the case of a new contact in which case it's stored in `contacts`, but never consumed.